package.xml

```diff
@@ -47,8 +47,9 @@
   <run_depend>roscpp</run_depend>
   <run_depend>roslaunch</run_depend>
   <run_depend>sensor_msgs</run_depend>
-  <!--run_depend>std_srvs</run_depend-->
   <run_depend>nodelet</run_depend>
+  <run_depend>dynamic_reconfigure</run_depend>
+  <run_depend>roslint</run_depend>
```

Comment:
@jonbinney doesn't look like it'll build without roslint as a run_depend:

`CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:224 (message):
  catkin_package() DEPENDS on the catkin package 'roslint' which must
  therefore be listed as a run dependency in the package.xml
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  pylon_camera/CMakeLists.txt:40 (catkin_package)
`